### PR TITLE
Feature/implement role access (basic)

### PR DESCRIPTION
**Pull request by Somiel**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -27,4 +27,24 @@ def admin_exists?
         self.type = "Client"
       end
     end
+
+    # Returns true if current user.type == "Admin", else returns false
+    def is_admin?
+      self.type == "Admin"
```
Comment (murderface13) [inline app/models/user.rb]: you can use self.is_a? Admin, i think

Comment (murderface13) [inline app/models/user.rb]: it will be more flexible, i suppose

Comment (Somiel) [inline app/models/user.rb]: mkay, i agree

File: app/models/user.rb [hunk 2]
```diff
@@ -27,4 +27,24 @@ def admin_exists?
...
+    end
+
+    # Returns true if current user.type == "Manager", else returns false
+    def is_manager?
```
Comment (murderface13) [inline app/models/user.rb]: and also you don't need to use 'is_' prefix(see ruby styleguide)

**Comment by murderface13** [comment]
:+1:

**Pull request merged**